vcfa/remove_leftovers_test.go

```diff
@@ -118,6 +118,26 @@ func removeLeftovers(govcdClient *govcd.VCDClient, verbose bool) error {
 		}
 	}
 
+	// --------------------------------------------------------------
+	// Regions
+	// --------------------------------------------------------------
+	if govcdClient.Client.IsSysAdmin {
+		regions, err := govcdClient.GetAllRegions(nil)
+		if err != nil {
+			return fmt.Errorf("error retrieving Regions: %s", err)
+		}
+		for _, region := range regions {
+			toBeDeleted := shouldDeleteEntity(alsoDelete, doNotDelete, region.Region.Name, "vcfa_region", 0, verbose)
```

Comment:
NIT: Output indent is a bit higher due to it depends on VC/NSX-T being available.

```suggestion
			toBeDeleted := shouldDeleteEntity(alsoDelete, doNotDelete, region.Region.Name, "vcfa_region", 1, verbose)
```